### DISCUSSION TITLE
Report "Group" or "Break" Headers

### DISCUSSION THREAD

**Discussion by bret-miller** [ariacom/Seal-Report#117] [Q&A] [answered]
I am trying to create a class roster report for school administration. Essentially, they want the report something like this:

Class name 1, Instructor
    Student, credit type, grade, completion date
    Student, credit type, grade, completion date
    Student, credit type, grade, completion date
Class name 2, Instructor
    Student, credit type, grade, completion date
    Student, credit type, grade, completion date
    Student, credit type, grade, completion date

The query returns the class name, instructor and student information as 1 row per student. How can I detect a change in class and output the header at the top of each group?

**Comment by ariacom** [comment, marked as answer]
In your model, you should just drop [Class name] and [Instructor] in the page panel,
then drop the other fields in the row panel.
